docs/setup-guide/copy_cert_between_edge_cluster.md

```diff
@@ -0,0 +1,91 @@
+#	File or Folder Copy Process Between Edge cluster(For Reference)
+
+## Abstract
+The purpose of this document is how to copy file or folder between machine A to machine B and C (virtual machine or cluster), and describe the each steps and copy process. This is reference documentation.
+If you have own way to copy, you can use your own way to copy and skip this doc.
+1. You can reference this document at [SSH login without password](http://www.linuxproblem.org/art_9.html).
+2. Generate public key in machine A if you want to copy A file or folder to machine B.
+3. Copy the public key in A to machine B and C.
+4. Copy ca, certs, and admin.conf process see detail 2.1
+
+## 1.1. Generate Key in machine A and copy pub key to machne B 
+-	Ubuntu 18.04, one for cloud-core, two for edge-core.
+
+### 1.1.1. Generate Key in root at machine A
+-	Run following command and keep "Enter" key, until key generated.
+```
+ssh-keygen -t rsa
+```
+Result:
+<img src="images/Key_01_generate.png" 
+     width="98%"  
+     align="center"/>
+
+### 1.1.2. Open pub key in B.
+```
+cat .ssh/id_rsa.pub
+```
+or
+```
+vi .ssh/id_rsa.pub
+```
+Result:
+<img src="images/Key_02_machineAkey.png" 
+     width="98%"  
+     align="center"/>
+
+### 1.1.3. Copy pub key to Machine B.
+Connect to machine B and opend .ssh/authorized_keys
+```
+vi .ssh/authorized_keys
+```
+you need append machine A public key to authorized_keys.
+Result:
+<img src="images/Key_03_machineBauthkey.png" 
+     width="98%"  
+     align="center"/>
+
+<img src="images/Key_04_machineBappend.png" 
+     width="98%"  
+     align="center"/>
+  
+### 1.1.4. Follow up 1.1.1 to 1.1.3 Copy Machine A pub key to Machine C.
+### 1.1.5. Follow up 1.1.1 to 1.1.3 Copy Machine B pub key to Machine C. (notes: we need copy machine B "admin.conf" to machine C).
```

Comment:
Please do not use "we".  It is better to write in the third person

Reply:
updated based on shaojun comment

Reply:
As Du Peng suggested, we don't use "you", the 2nd person. Please use the third person which is a common practice for narrative.